plugin/core/configurations.py

```diff
@@ -53,12 +53,12 @@ def match_view(self, view: sublime.View, include_disabled: bool = False) -> Gene
         except (IndexError, RuntimeError):
             pass
 
-    def update(self, config_name: Optional[str] = None) -> None:
+    def update(self, updated_config: Optional[str] = None) -> None:
```

Comment:
I would assume that `updated_config` is an object until I look at the type
... `updated_config_name` would me think that it is a string